docs/content/preview/reference/configuration/insecure-yugabyted-examples.md

```diff
@@ -0,0 +1,939 @@
+
+-----
+
+### demo
+
+Use the `yugabyted demo` command to use the demo [Northwind sample dataset](../../../sample-data/northwind/) with YugabyteDB.
+
+#### Commands
+
+The following subcommands are available for the `yugabyted demo` command:
+
+- [connect](#connect-1)
+- [destroy](#destroy-1)
+
+#### Syntax
+
+```sh
+Usage: yugabyted demo [command] [flags]
+```
+
+#### connect
+
+Use the `yugabyted demo connect` subcommand to load the  [Northwind sample dataset](../../../sample-data/northwind/) into a new `yb_demo_northwind` SQL database, and then open the `ysqlsh` prompt for the same database.
+
+#### Syntax
+
+```sh
+Usage: yugabyted demo connect [flags]
+```
+
+#### Flags
+
+-h | --help
+: Print the help message and exit.
+
+--config *config-file*
+: The path to the configuration file of the yugabyted server whose logs are desired.
+
+--data_dir *data-directory*
+: The data directory for the yugabyted server whose logs are desired.
+
+--base_dir *base-directory*
+: The base directory for the yugabyted server whose logs are desired.
+
+--log_dir *log-directory*
+: The directory to store yugabyted logs. Must be an absolute path. This flag controls where the logs of the YugabyteDB nodes are stored.
+
+#### destroy
+
+Use the `yuagbyted demo destroy` subcommand to shut down the yugabyted single-node cluster and remove data, configuration, and log directories. This subcommand also deletes the `yb_demo_northwind` database.
+
+#### Syntax
+
+```sh
+Usage: yugabyted demo destroy [flags]
+```
+
+#### Flags
+
+-h | --help
+: Print the help message and exit.
+
+--config *config-file*
+: The path to the configuration file of the yugabyted server whose logs are desired.
+
+--data_dir *data-directory*
+: The data directory for the yugabyted server whose logs are desired.
+
+--base_dir *base-directory*
+: The base directory for the yugabyted server whose logs are desired.
+
+--log_dir *log-directory*
+: The directory to store yugabyted logs. Must be an absolute path. This flag controls where the logs of the YugabyteDB nodes are stored.
+
+-----
+
+## Environment Variables
+
+In the case of multi-node deployments, all nodes should have similar environment variables.
+
+Changing the values of the environment variables after the first run has no effect.
+
+### YSQL
+
+Set `YSQL_PASSWORD` to use the cluster in enforced authentication mode.
+
+Combinations of environment variables and their uses.
+
+- `YSQL_PASSWORD`
+
+  Update the default yugabyte user's password.
+
+- `YSQL_PASSWORD, YSQL_DB`
+
+  Update the default yugabyte user's password and create `YSQL_DB` named DB.
+
+- `YSQL_PASSWORD, YSQL_USER`
+
+  Create `YSQL_USER` named user and DB with password `YSQL_PASSWORD`.
+
+- `YSQL_USER`
+
+  Create `YSQL_USER` named user and DB with password `YSQL_USER`.
+
+- `YSQL_USER, YSQL_DB`
+
+  Create `YSQL_USER` named user with password `YSQL_USER` and `YSQL_DB` named DB.
```

Comment:
Shouldn't it be

Create `YSQL_USER` named user for `YSQL_DB` named DB.

Reply:
Keeping as it is for now.